private/bufpkg/bufmodule/uploader.go

```diff
@@ -65,6 +91,16 @@ type UploadOptions interface {
 	//
 	// Will always be present if CreateIfNotExist() is true.
 	CreateModuleVisibility() ModuleVisibility
+	// Tags returns unique and sorted set of tags to be added as labels.
+	//
+	// We disallow the setting of `--tag` when uploading a workspace (e.g. a ModuleSet with
+	// 1+ target Modules), so we need to separate this out to handle in Upload.
+	Tags() []string
+	// BranchOrDraft returns a branch/draft to be set as a label.
```

Comment:
Same comment re: differences with labels as in above on tag. Additionally, explaining why branch and draft are using the same field here for future readers.